webapp/src/Serializer/TeamAffiliationVisitor.php

```diff
@@ -99,7 +107,16 @@ public function onPostSerialize(ObjectEvent $event): void
                 'logo',
                 null
             );
-            $visitor->visitProperty($property, [['href' => $route, 'mime' => mime_content_type($affiliationLogo), 'width' => $imageSize[0], 'height' => $imageSize[1]]]);
+            $visitor->visitProperty($property, [
+                [
+                    'href'     => $route,
+                    'mime'     => mime_content_type($affiliationLogo),
+                    'width'    => $imageSize[0],
+                    'height'   => $imageSize[1],
+                    'hash'     => md5_file($affiliationLogo),
```

Comment:
how expensive is it to do this again and again?

Reply:
🤷‍♂️ files should not be gigantic right? So I think it’s not too bad

Reply:
Having said that, this field is optional so if you feel it might be (too) expensive, we can drop it.
Alternatively we can use Symfony's `CacheInterface` to cache it, so that we only calculate it once. However, that does mean people need to clear the Symfony cache when they change an image. We can automatically do this when you upload one in the jury interface, but not when manually placing a file in the directory. Unless we can come up with a cache key that identifies the file uniquely (mtime might work but not ideal).

So maybe we just drop it?

Reply:
I'm in favor of dropping

Reply:
OK let me drop it while it's hot

Reply:
Aaaannnnndddd it's gone.